packages/pyright-internal/src/tests/textRange.test.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I'm not sure we need a test for this. If this functionality doesn't work, it will be pretty obvious immediately. Each new test adds time and compute for every subsequent change, so I try to add tests only for things that have the potential for regressions that would otherwise go undetected. It's fine to leave this test here, but something to keep in mind for the future.